### PR TITLE
changed the github link of Meshmap & Meshery Cloud

### LINKED ISSUE #4634: Meshary cloud github link not found
### Description

In this [page](https://layer5.io/community/handbook/repository-overview), github link got 404 error on Meshary cloud row.
### Expected Behavior

I think we can this [https://github.com/meshery/meshery](https://github.com/meshery/meshery) to that github icon link
### Screenshots

Screenshot from 2023-07-27 10-58-55-5

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by SNIPACER**
**Description**
I have changed the GH links of MeshMap & Meshery Cloud in the Community/handbook/repository-overview
This PR fixes #4634 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit cc21eb21db356fbd6898f16acb6560190d109ca5 at: https://64c2b7be4f8c9204cb7bddb2--layer5.netlify.app

**Comment by Chadha93** [comment]
Hi @SNIPACER, thanks for opening a PR, but another update was made with a different fix for this PR - https://github.com/layer5io/layer5/pull/4643. You can close out this PR.

I'll make sure to get you a new issue assigned.

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ export const repo_data = [
         language: "React.js",
         maintainers_name: ["Pranav Singh","Uzair Shaikh"],
         link: ["#"],
-        repository: "https://github.com/layer5io/meshery-cloud",
+        repository: "https://github.com/meshery/meshery",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:126]:
```suggestion
        repository: "https://github.com/layer5io/meshery-cloud",
```

**Review by leecalcote — APPROVED** [review]
Thanks!

**Pull request merged**